### PR TITLE
Shoji's change

### PR DESCRIPTION
**Pull request by shojikajita**
Please accept my pull-request!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,6 +21,9 @@
 <p><img width="30%" height="30%" src="https://upload.wikimedia.org/wikipedia/commons/1/15/Red_Apple.jpg"/>
 </p>
 
+<p><img width="30%" height="30%" src="https://twitter.com/shojikajita/status/636858594311311360/photo/1"/>
```
Comment (bkirschn) [inline index.html]: This is a great change!

Comment (shojikajita) [inline index.html]: I think so too!

**Comment by naoshiraoka** [comment]
hello

**Comment by miyazakima** [comment]
comment

**Pull request merged**